app/components/hiring_staff/vacancies_component.rb

```diff
@@ -45,7 +45,7 @@ def vacancy_type_tab_link(vacancy_type)
   def set_vacancies
     @vacancies =
       if @filters[:managed_school_ids]&.any?
-        Vacancy.in_organisation_ids(@filters[:managed_school_ids])
+        Vacancy.in_organisation_ids(@filters[:managed_school_ids]).distinct
```

Comment:
I would put `.distinct` into the scope
```
scope :in_organisation_ids, -> (ids) do
    joins(:organisation_vacancies).where(organisation_vacancies: { organisation_id: ids }).distinct
  end
```
So that we do not have to put it everywhere